### PR TITLE
Description for downloading CrowPi Image updated

### PR DESCRIPTION
**Pull request by DieterHolz**

### CODE REVIEW

File: content/german/setup/raspberry.md [hunk 1]
```diff
@@ -17,12 +17,10 @@ Anleitungen zur Installation sind ebenfalls auf der genannten Homepage verfügba
...
 
-Das Image für den CrowPi, welches das Betriebssystem für den Raspberry PI beinhaltet kann direkt aus dem Github
-Repository des CrowPi Projekts bezogen werden. Das neueste Release des Betriebssystems findet ihr unter diesem Link:
-[Download CrowPi Image]({{% siteparam "repoURL" %}}/releases/latest)
+Das Image für den CrowPi, welches das Betriebssystem für den Raspberry PI beinhaltet, kann direkt aus dem [Github
+Repository des Pi4J Projekts](https://github.com/Pi4J/pi4j-os) bezogen werden, das auch noch weitere spezialisierte Images bereitstellt.
 
-Bei GitHub kann das Image dann hier heruntergeladen werden:
-{{< img alt="GitHub Download CrowPi Image" src="setup/download-crowpi-image.JPG" >}}
+Das neueste Release des Betriebssystems für den CrowPi findet ihr unter diesem Link: [Download CrowPi Image](https://pi4j-download.com/main-crowpi.img.zip)
```
Comment (ppmathis) [inline content/german/setup/raspberry.md:23]:
```suggestion
Das neueste Release des Betriebssystems für den CrowPi findet ihr unter diesem Link: [Download CrowPi Image](https://pi4j-download.com/latest.php?flavor=crowpi)
```

Same change as in https://github.com/Pi4J/pi4j-example-crowpi/pull/18

File: content/german/setup/raspberry.md [hunk 2]
```diff
@@ -17,12 +17,10 @@ Anleitungen zur Installation sind ebenfalls auf der genannten Homepage verfügba
 
 ### 2. Herunterladen des FHNW CrowPi Image
 
-Das Image für den CrowPi, welches das Betriebssystem für den Raspberry PI beinhaltet kann direkt aus dem Github
-Repository des CrowPi Projekts bezogen werden. Das neueste Release des Betriebssystems findet ihr unter diesem Link:
-[Download CrowPi Image]({{% siteparam "repoURL" %}}/releases/latest)
+Das Image für den CrowPi, welches das Betriebssystem für den Raspberry PI beinhaltet, kann direkt aus dem [Github
```
Comment (ppmathis) [inline content/german/setup/raspberry.md:20]:
```suggestion
Das Image für den CrowPi, welches das Betriebssystem für den Raspberry Pi beinhaltet, kann direkt aus dem [Github
```

File: content/german/setup/raspberry.md [hunk 3]
```diff
@@ -103,14 +101,29 @@ eine Verbindung via Kabel aufgebaut werden.
...
+Der Raspberry Pi und der Laptop müssen in der Regel mit dem gleichen WLAN verbunden sein. 
+
+#####  Verwenden des Pi4J-Spot
+
+Auf dem CrowPi-Image ist eine WLAN-Verbindung voreingestellt.
+
+- ssid: ``Pi4J-Spot``
+- password: ``MayTheCodeBeWithYou!``
+
+Sobald ein dementsprechender Hotspot, zum Beispiel auf einem Smartphone, angelegt wird, verbindet sich der CrowPi mit diesem Hotspot und die aktuelle IP-Nummer des CrowPi wird als Hintergrundbild angezeigt.
```
Comment (ppmathis) [inline content/german/setup/raspberry.md:113]:
```suggestion
Sobald ein dementsprechender Hotspot, zum Beispiel auf einem Smartphone, angelegt wird, verbindet sich der CrowPi mit diesem Hotspot und die aktuelle IP-Adresse des CrowPi wird als Hintergrundbild angezeigt. 
```

To be consistent with the paragraph written slightly further down below.

**Review by ppmathis — CHANGES_REQUESTED** [review]
Looks good to me! I only have a few suggestions and also updated the download URL here to point to the newly created helper.

**Review by DieterHolz — COMMENTED** [review]
Fine.

**Review by ppmathis — APPROVED** [review]
LGTM.

**Pull request merged**